ccmlib/node.py

```diff
@@ -556,6 +559,40 @@ def watch_log_for(self, exprs, from_mark=None, timeout=600, process=None, verbos
                         if process.returncode == 0:
                             return None
 
+    def watch_log_for_no_errors(self, exprs, from_mark=None, timeout=600, process=None, verbose=False, filename='system.log'):
+        """
+        Watch the log until one or more (regular) expression are found.
+        This methods when all the expressions have been found or the method
+        timeouts (a TimeoutError is then raised). On successful completion,
+        a list of pair (line matched, match object) is returned.
+
+        This method is different from watch_log_for since it will fail if
```

Comment:
fail Cassandra

Reply:
we don't stop cassandra, we only stop watching the logs.

Reply:
True, my point is this fail doesn't tell me what failure we talk about :-)

Reply:
How about

> This method is different from watch_log_for as it will raise a AssertionError if the logs contain an error; this assertion will contain the errors found in the logs.